src/beamlime/offline/fake_2d_image.ipynb

```diff
@@ -0,0 +1,272 @@
+{
```

Comment:
I... still didn't set up the document pages so here is the exported pdf 

[fake_2d_image.pdf](https://github.com/scipp/beamlime/files/11052385/fake_2d_image.pdf)